addons/pointing/XEH_postInit.sqf

```diff
@@ -0,0 +1,9 @@
+#include "script_component.hpp"
+
+#include "initKeybinds.sqf"
+
+if (IS_MOD_LOADED(diwako_dui_radar)) then {
```

Comment:
Wouldn't it be better to include in common preInit just like you did with `ace_finger`?

Reply:
I just assessed that it might not be so useful for other components as ACE stuff. Should I move it?

Reply:
You might be right. It can be done later if needed anyway.